sky/shell/platform/android/io/flutter/view/FlutterMain.java

```diff
@@ -277,22 +331,69 @@ public Binding connectToService(FlutterView view, Core core, MessagePipeHandle p
         });
     }
 
+    /**
+     * Initialize our Flutter config values by searching for a .properties file
+     * containing properties for our configuration variables. If no such
+     * properties file is found in our assets folder, the configuration
+     * variables will retain their default values.
```

Comment:
Internal use case - can discuss offline.

Regarding performance, I just ran a bunch of benchmarks, and it looks like the only substantive call in here is `AssetManager#list(String)` - sans that call, this runs in under a millisecond.  Also, it looks like that call is not cached at the Android system level, so subsequent calls will be just as expensive (~30ms).

Given that we call `AssetManager#list(String)` in `#initAot(Context)` as well, it may be worth caching the asset list in `#startInitialization(Context)` and passing it down to the other init methods.  Then we'll incur the hit only once, and it's a hit we were taking anyway.

Thoughts?